sycl/include/sycl/ext/oneapi/matrix/matrix-unified.hpp

```diff
@@ -112,6 +112,31 @@ get_wi_data(Group sg, joint_matrix<Group, T, Use, Rows, Cols, Layout> &jm) {
 #endif // defined(__SYCL_DEVICE_ONLY__)
 }
 
+template <typename Group, typename T, use Use, size_t M, size_t N,
+          layout Layout, typename F>
+inline __SYCL_ALWAYS_INLINE void
+joint_matrix_apply(Group sg, joint_matrix<Group, T, Use, M, N, Layout> &jm,
```

Comment:
please handle unused parameter

Reply:
unused sg and jm on the host side. Please use same thing like: std::ignore = sg;